### DISCUSSION TITLE
version 2.0.9 breaks pandasql

### DISCUSSION THREAD

**Discussion by rggeorge** [sqlalchemy/sqlalchemy#9673] [Usage Questions] [answered]
Find any example of how to use python `pandasql`, or use this one, and it won't run correctly after 2.0.9:

```
df = pd.DataFrame(data=[1,2,3], columns=['col'])
import pandasql as ps
sqlcode = '''select * from df'''
newdf = ps.sqldf(sqlcode)
```

Error: `ObjectNotExecutableError: Not an executable object: 'select * from df'`

Issue is fixed by installing `sqlalchemy<2.0`

1. I don't personally care, just trying to be a good citizen and report issues
2. I'm not sure if it was the 2.0.9 update or an earlier one, might have been as early as 2.0.0.

**Comment by zzzeek** [comment]
plain strings in SQLAlchemy are usually embedded in text() and this is required in 2.0:  https://docs.sqlalchemy.org/en/20/changelog/migration_20.html#execute-method-more-strict-execution-options-are-more-prominent 

there is no change to this in 2.0.9.   

If this is part of pandas' usage contract you should [report it to pandas](https://github.com/pandas-dev/pandas/issues/51015).

**Comment by gordthompson** [comment, marked as answer]
This has already been reported to pandasql

https://github.com/yhat/pandasql/issues/99
